packages/node/src/client.ts

```diff
@@ -138,6 +147,46 @@ export class NodeClient extends BaseClient<NodeClientOptions> {
     );
   }
 
+  /**
+   * Create a cron monitor check in and send it to Sentry.
+   *
+   * @param checkIn An object that describes a check in.
+   * @param upsertMonitorConfig An optional object that describes a monitor config. Use this if you want
+   * to create a monitor automatically when sending a check in.
+   */
+  public captureCheckIn(checkIn: CheckIn, monitorConfig?: MonitorConfig): void {
+    if (!this._isEnabled()) {
+      __DEBUG_BUILD__ && logger.warn('SDK not enabled, will not capture checkin.');
+      return;
+    }
+
+    const options = this.getOptions();
+    const { release, environment, tunnel } = options;
+
+    const serializedCheckIn: SerializedCheckIn = dropUndefinedKeys({
+      check_in_id: uuid4(),
+      monitor_slug: checkIn.monitorSlug,
+      status: checkIn.status,
+      duration: checkIn.duration,
+      release,
+      environment,
+    });
+
+    if (monitorConfig) {
+      serializedCheckIn.monitor_config = dropUndefinedKeys({
+        schedule: monitorConfig.schedule,
+        checkin_margin: monitorConfig.checkinMargin,
+        max_runtime: monitorConfig.maxRuntime,
+        timezone: monitorConfig.timezone,
+      });
+    }
+
+    const envelope = createCheckInEnvelope(serializedCheckIn, this.getSdkMetadata(), tunnel, this.getDsn());
+
+    __DEBUG_BUILD__ && logger.warn('Sending checkin: ', checkIn);
```

Comment:
l: In node this might not get printed pretty if `checkIn` is nested. Should we maybe even log `serializedCheckIn` for convenience?

Reply:
Ah we should not even have this tbh, that was left over from our previous PR.

Reply:
af4a4fb9687fc6c18732fa4af077f9fabb3ccbf2